### PR TITLE
Add a setup.py 

### PR DESCRIPTION
**Pull request by marwahaha**
Resolves issue #34

cc @jpurviance

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+"""
+
+from setuptools import setup, find_packages
+
+setup(
+	name='attending',
+	version='0.0.1',
+	description='A manager for offline documentation',
+	url='https://github.com/ivanov/attending',
+	packages=find_packages(exclude=['fizbuz', 'foobar', 'latest', 'prior-work', 'tests', 'versionless']),
```
Comment (marwahaha) [inline setup.py]: Perhaps we can fix this list after the folders move.

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,4 @@
+## Local install
```
Comment (marwahaha) [inline .gitignore:1]: Is this the right way to comment a python file?

**Comment by ivanov** [comment]
lol, ooops, #38 also does something similar.

File: setup.py (hunk 1 above)
Comment (ivanov) [reply setup.py]: you can also do `packages=['attending']` and skip find_packages altogether.

**Pull request merged**